data/transactions/logic/README.md

```diff
@@ -275,6 +277,7 @@ Some of these have immediate data in the byte or bytes after the opcode.
 | 53 | GlobalNumByteSlice | uint64 | Number of global state byteslices in ApplicationCall. LogicSigVersion >= 3. |
 | 54 | LocalNumUint | uint64 | Number of local state integers in ApplicationCall. LogicSigVersion >= 3. |
 | 55 | LocalNumByteSlice | uint64 | Number of local state byteslices in ApplicationCall. LogicSigVersion >= 3. |
+| 56 | ExtraProgramPages | uint64 | Number of additional pages for each of the application's approval and clear state programs. An ExtraProgramPages of 1 means 2048 more total bytes, or 1024 for each program. LogicSigVersion >= 4. |
```

Comment:
@jannotti , should it be LogicSigVersion >= 3? that's the value in langSpec file.

Reply:
we enable it [along with teal v4](https://github.com/algorand/go-algorand/pull/2255/files#diff-a75ef7294adbd7ae646c49ae2a518253936983c0a16594b84f20f15403767a2fR935) so this comment looks correct. 

Additionally I have that is version 4. So the lang spec is incorrect.
```{AppProgramExtraPages, StackUint64, 4}```